Source/Core/DolphinQt2/QtUtils/ElidedButton.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QPushButton>
+
+class ElidedButton : public QPushButton
+{
+public:
+  ElidedButton(const QString& text = QStringLiteral(""),
+               Qt::TextElideMode elide_mode = Qt::ElideRight);
+  Qt::TextElideMode elideMode() const;
```

Comment:
These should follow the coding style for function naming where applicable (i.e. `ElideMode()`, etc)